test/test.py

```diff
@@ -306,14 +308,21 @@ def test_ontology_label_field(self):
         self.assertEqual(protocol.nucleic_acid_source, 'single cell')
 
     def test_accessions_fields(self):
+        self.expected_accessions = {
```

Comment:
All expectation are passed as arguments, why does this need one need to be set as an attribute?